Content.Server/Administration/Systems/AdminVerbSystem.cs

```diff
@@ -388,9 +391,30 @@ private void AddAdminVerbs(GetVerbsEvent<Verb> args)
                         },
                         Icon = new SpriteSpecifier.Rsi(new ResPath("/Textures/Interface/Actions/actions_borg.rsi"), "state-laws"),
                     });
+                }
+				
+				// thaven moods
```

Comment:
mixing tabs + use `// Begin DeltaV Additions - thaven moods` and `// End DeltaV Additions`